src/client/game.cpp

```diff
@@ -1287,7 +1287,8 @@ void Game::shutdown()
 		formspec->quitMenu();
 
 #ifdef HAVE_TOUCHSCREENGUI
-	g_touchscreengui->hide();
+	if (g_touchscreengui->isInitialized())
```

Comment:
can't you just make `hide()` check this? the caller shouldn't have to care

Reply:
Sure, done. Did the same to other public TouchScreenGUI methods with segfault potential.